frontend/src/pages/PropertyDetails.jsx

```
@@ -0,0 +1,403 @@
+        }));
+    };
+
+    const handleBooking = () => {
+        console.log('Booking with details:', bookingDates);
+        // Add booking functionality here
+    };
+
+    if (loading) return (
+        <div className="min-h-screen flex justify-center items-center bg-gray-50">
+            <div className="text-center">
+                <div className="animate-spin rounded-full h-12 w-12 border-b-2 border-blue-600 mx-auto mb-4"></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)